core/src/test/java/kafka/admin/ConfigCommandTest.java

```diff
@@ -1670,80 +1205,6 @@ private ConfigCommand.ConfigCommandOptions createOpts(String entityType, Optiona
         return new ConfigCommand.ConfigCommandOptions(toArray(optArray, nameArray, otherArgs));
     }
 
-    private void checkEntity(String entityType, Optional<String> entityName, String expectedEntityName, List<String> otherArgs) {
-        ConfigCommand.ConfigCommandOptions opts = createOpts(entityType, entityName, otherArgs);
-        opts.checkArgs();
-        ConfigCommand.ConfigEntity entity = ConfigCommand.parseEntity(opts);
```

Comment:
`parseEntity` is unused now, could you please remove it?

Reply:
Thanks for your review! @chia7712, but parseEntity had already been removed in this commit, is there anything else I should do?

Reply:
Thank you for your review @chia7712, parseEntity had been removed.